src/commands.cpp

```diff
@@ -7358,24 +7407,32 @@ CommandMediaCodecs::CommandMediaCodecs(MegaClient* c, Callback cb)
 }
 
 void CommandMediaCodecs::procresult()
-{
-    int version = 0;
-    if (client->json.isnumeric())
+{    
+    Error errorDetails;
+    int64_t result;
```

Comment:
I think we can simply keep the `int version`. Is there any specific reason for changing the data type?

Reply:
None specific, only define the same type that is returned by `json.getint()`